pkg/test/utils/test_context.go

```diff
@@ -43,12 +43,13 @@ type KBTestContext struct {
 	Resources  string
 	ImageName  string
 	Kubectl    *Kubectl
+	BinName    string
```

Comment:
Can we please use the full var name here like we're doing for other fields? e.g `ImageName` or `TestSuffix`.
That's more readable imo.
```suggestion
	BinaryName    string
```